doc/devel/coding_guide.rst

```diff
@@ -439,21 +441,30 @@ Other branches are fed through :ref:`automatic <automated-backports>` or
 targeting other branches is only rarely necessary for special maintenance
 work.
 
-.. backport_strategy:
+.. _backport-strategy:
 
 Backport strategy
 -----------------
 
-We will always backport to the patch release branch (*v3.N.x*):
+Backports to the patch release branch (*v3.N.x*) are the changes that will be
+included in the next patch (aka bug-fix) release.  The goal of the patch
+releases is to fix bugs introduced in the associated minor release without
+adding any new regressions or behavior changes.  We will always attempt to
+backport:
 
 - critical bug fixes (segfault, failure to import, things that the
   user can not work around)
-- fixes for regressions against the last two releases.
+- fixes for regressions against the last two minor releases.
 
-Everything else (regressions against older releases, bugs/api
-inconsistencies the user can work around in their code) are on a
-case-by-case basis, should be low-risk, and need someone to advocate
-for and shepherd through the backport.
+Other changes, such as regressions against older releases or api
+inconsistencies the user can work around in their code, are on a case-by-case
+basis, should be low-risk, and need someone to advocate for and shepherd
+through the backport.
```

Comment:
This bit about advocating for a backport is not something I’ve observed in the admittedly short time I’ve been here.  I think we more often state why we’re _not_ backporting something.